tools-local/Microsoft.ML.StableApi/Microsoft.ML.StableApi.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <!-- needs to contain all frameworks which src projects wish to restore -->
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.ML" Version="1.0.0-preview-27630-5"/>
+    <PackageReference Include="Microsoft.ML.DataView" Version="1.0.0-preview-27630-5"/>
+    <PackageReference Include="Microsoft.ML.CpuMath" Version="1.0.0-preview-27630-5"/>
+    <PackageReference Include="Microsoft.ML.FastTree" Version="1.0.0-preview-27630-5"/>
+    <PackageReference Include="Microsoft.ML.LightGbm" Version="1.0.0-preview-27630-5"/>
+    <PackageReference Include="Microsoft.ML.ImageAnalytics" Version="1.0.0-preview-27630-5"/>
+    <PackageReference Include="Microsoft.ML.Mkl.Components" Version="1.0.0-preview-27630-5"/>
+    <PackageReference Include="Microsoft.ML.Mkl.Redist" Version="1.0.0-preview-27630-5"/>
```

Comment:
The Redist package doesn't have public APIs. Do we need to include it here?

Reply:
Yes you are right I'll remove that.

---
In reply to: [280104238](https://github.com/dotnet/machinelearning/pull/3623#discussion_r280104238) [](ancestors = 280104238)